public/stylesheets/style.css

```diff
@@ -1,9 +1,32 @@
 body {
   padding: 50px;
   font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
-  background-color: green;
+  background-color: DeepSkyBlue;
```

Comment:
I don't like the background color. It's TOO blue. I want a light blue. Love the hover option though.

Reply:
-